http/server.go

```diff
@@ -559,6 +561,14 @@ func (h *DrandHandler) ChainHashes(w http.ResponseWriter, r *http.Request) {
 	_, _ = w.Write(b)
 }
 
+//nolint:gomnd
+func (h *DrandHandler) HandleUnderflow(w http.ResponseWriter, r *http.Request) {
+	time.Sleep(29 * time.Second)
```

Comment:
It seems this didn't work, although the clients eventually stopped hammering us.
Either they timed out before 29s, or they got the beacon and it didn't verify.
If they had timed out (or waited 29s) though, we wouldn't expect to have seen as many requests as we did

Reply:
Yeah, this is really strange :/ And the error page TTL is set to 10, so it isn't just Cloudfront caching the errors.